index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="styles1.css" rel="stylesheet">
+        <title>Microverse First Project</title>
+    </head>
+    <body>
+        <header>
+            <nav class="toolbar">
+                <p class="text1">My Logo</p>
+                <p class="nav"><button id="menu" type="button"><img src="Union.png" alt=""></button></p>
+            </nav>
+        </header>
+        
+        <div class="container">
+            <div class="headline">
+                <h1>I’m Tanmay Glad to see you!</h1>
+                <p> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <p>LETS CONNECT</p>
+                <ul>
+                    <li><a href="#"><img src="Vector.svg" alt=""></a></li>
+                    <li><a href="#"><img src="Linkedinicon.svg" alt=""></a></li>
+                    <li><a href="#"><img src="Vector(1).svg" alt=""></a></li>
+                    <li><a href="#"><img src="Vector(2).svg" alt=""></a></li>
+                    <li><a href="#"><img src="Vector(3).svg" alt=""></a></li>
+                </ul>
```

Comment:
- [ ] You have done a tremendous job on converting the required figma design! I therefore highly recommend that you continue to add more effort in styling your design to match the other one.

| Your design | Figma design |
| --- | --- |
| <img width="309" alt="real2" src="https://user-images.githubusercontent.com/4247740/143578416-5b5091ac-cc67-4ecc-89b1-eddbc2986114.PNG"> | <img width="284" alt="real" src="https://user-images.githubusercontent.com/4247740/143578392-edae560e-df59-442a-84ab-b6d87d390d58.PNG"> |
| - Compare the font color and correct it | ✅  |
| - Also checkout the icons alignment and match them with the design on the right. | ✅  |

Reply:
Added the necessary changes